frontend/src/pages/item/_itemId/index.vue

```diff
@@ -246,94 +246,62 @@ import { getItemDetailsLink, getMediaStreams } from '@/utils/items';
 import { getItemizedSelect } from '@/utils/forms';
 import { useRemote } from '@/composables';
 
-export default defineComponent({
-  async setup() {
-    const { params } = useRoute();
-    const itemId = params.itemId;
-    const remote = useRemote();
-    const item = (
+const route = useRoute();
+const remote = useRemote();
+
+const item = ref<BaseItemDto>({});
+const currentSource = ref<MediaSourceInfo>({});
+const currentVideoTrack = ref<number>();
+const currentAudioTrack = ref<number>();
+const currentSubtitleTrack = ref<number>();
+
+const crew = computed((): BaseItemPerson[] => {
+  if (!item.value?.People) {
+    return [];
+  }
+
+  return item.value.People.filter((person: BaseItemPerson) =>
+    ['Director', 'Writer'].includes(person?.Type ?? '')
+  );
+});
```

Comment:
I think this can be written as

```suggestion
const crew = computed(
  (): BaseItemPerson[] =>
    item.value?.People?.filter((person: BaseItemPerson) =>
      ['Director', 'Writer'].includes(person?.Type ?? '')
    ) ?? []
);
```